app/javascript/components/ProductEdit/ProductTab/PriceEditor.tsx

```diff
@@ -48,11 +49,16 @@ export const PriceEditor = ({
         onChange={(newAmount) => setPriceCents(newAmount ?? 0)}
         currencyCodeSelector={currencyCodeSelector}
       />
+      {isFreeProduct && (
+        <div role="alert" className="info">
+          Free products require a pay what they want price.
+        </div>
+      )}
       <Details
         className="toggle"
         open={isPWYW}
         summary={
-          <Toggle value={isPWYW} onChange={setIsPWYW}>
+          <Toggle value={isPWYW || isFreeProduct} onChange={setIsPWYW} disabled={isFreeProduct}>
```

Comment:
```suggestion
          <Toggle value={isPWYW} onChange={setIsPWYW} disabled={isFreeProduct}>
```

We shouldn't need this if PWYW setting is being enabled correctly, right? (better practice to keep value/onChange using a simple state value IMO)